org.eclipse.winery.topologymodeler.ui/src/doc/lib/README.md

```diff
@@ -0,0 +1,85 @@
+# @Winery/Topologymodeler
+This makes the [OpenTOSCA/Winery topologymodeler](https://github.com/OpenTOSCA/winery) module available as npm package.
+
+## Install
+### `npm install @winery/topologymodeler --save`
+
+## Usage
+### 1. Add the `WineryModule` to your app.module.ts:
+```
+import { WineryModule } from '@winery/topologymodeler';
+
+...
+
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  imports: [
+    BrowserModule,
+    ...,
+    WineryModule
+  ],
+  providers: [],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+
+### 2. Use Topologymodeler to render a TopologyTemplate:
+```
+<winery-topologymodeler [topologyModelerData]="...">
+</winery-topologymodeler>
+```
+
+#### `topologyModelerData` has to be of this type:
+```
+export interface TopologyModelerInputDataFormat {
+        configuration: {
+            readonly: boolean,
+            endpointConfig: TopologyModelerConfiguration
+        };
+        topologyTemplate: TTopologyTemplate;
+        visuals: Visuals;
+}
+
+TopologyModelerConfiguration {
+    readonly id: string;
+    readonly ns: string;
+    readonly repositoryURL: string;
+    readonly uiURL: string;
+    readonly compareTo: string;
+}
+```
+
+### Attention: this package has the following `peerDependencies`:
+##### Please make sure that you install them manually or by adding them to your `dependencies`.
```

Comment:
This is not a heading. Just print it in bold using `**text**`.

Reply:
done